MrE/build.gradle

```diff
@@ -53,6 +53,9 @@ def includeDesktopSupport = true
 // Defining my dependencies. In this case, WPILib (+ friends), and vendor libraries.
 // Also defines JUnit 5.
 dependencies {
+	// https://mvnrepository.com/artifact/org.apache.commons/commons-lang3
+	implementation group: 'org.apache.commons', name: 'commons-lang3', version: '3.12.0'
```

Comment:
I ended not using the Pair for Apache Commons but here is an example of how to import a 3rd party library.